backend/de.metas.edi/src/main/sql/postgresql/system/80-de.metas.edi/5736690_sys_gh19102_Invoice_IsEdiEnabled.sql

```diff
@@ -0,0 +1,162 @@
+-- Column: C_Invoice.IsEdiEnabled
+-- 2024-10-10T16:40:41.807Z
+UPDATE AD_Field SET DisplayLogic='@DocBaseType@=''ARC''',Updated=TO_TIMESTAMP('2024-10-10 19:40:41','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=551567
+;
+
+-- Field: Rechnung -> Rechnung -> Beleg soll per EDI übermittelt werden
+-- Column: C_Invoice.IsEdiEnabled
+-- Field: Rechnung(167,D) -> Rechnung(263,D) -> Beleg soll per EDI übermittelt werden
+-- Column: C_Invoice.IsEdiEnabled
+-- 2024-10-10T16:41:58.560Z
+UPDATE AD_Field SET DisplayLogic='@DocBaseType@=''ARC'' & @DocSubType@=''CS''',Updated=TO_TIMESTAMP('2024-10-10 19:41:58','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=551567
+;
+
+-- UI Element: Rechnung -> Rechnung.Beleg soll per EDI übermittelt werden
+-- Column: C_Invoice.IsEdiEnabled
+-- UI Element: Rechnung(167,D) -> Rechnung(263,D) -> advanced edit -> 10 -> advanced edit.Beleg soll per EDI übermittelt werden
+-- Column: C_Invoice.IsEdiEnabled
+-- 2024-10-10T16:43:05.271Z
+UPDATE AD_UI_Element SET AD_UI_ElementGroup_ID=541214, SeqNo=100,Updated=TO_TIMESTAMP('2024-10-10 19:43:05','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_UI_Element_ID=626188
+;
+
+-- UI Element: Rechnung -> Rechnung.Beleg soll per EDI übermittelt werden
+-- Column: C_Invoice.IsEdiEnabled
+-- UI Element: Rechnung(167,D) -> Rechnung(263,D) -> advanced edit -> 10 -> advanced edit.Beleg soll per EDI übermittelt werden
+-- Column: C_Invoice.IsEdiEnabled
+-- 2024-10-10T16:43:32.233Z
+UPDATE AD_UI_Element SET IsAdvancedField='Y',Updated=TO_TIMESTAMP('2024-10-10 19:43:32','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_UI_Element_ID=626188
+;
+
+-- UI Element: Rechnung -> Rechnung.Beleg soll per EDI übermittelt werden
+-- Column: C_Invoice.IsEdiEnabled
+-- UI Element: Rechnung(167,D) -> Rechnung(263,D) -> advanced edit -> 10 -> advanced edit.Beleg soll per EDI übermittelt werden
+-- Column: C_Invoice.IsEdiEnabled
+-- 2024-10-10T16:43:39.777Z
+UPDATE AD_UI_Element SET SeqNo=35,Updated=TO_TIMESTAMP('2024-10-10 19:43:39','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_UI_Element_ID=626188
+;
+
+-- Column: C_Doc_Outbound_Log.IsEdiEnabled
+-- Column SQL (old): (select bp.IsEdiInvoicRecipient from C_BPartner bp where bp.C_BPartner_ID=C_Doc_Outbound_Log.C_BPartner_ID)
+-- Column: C_Doc_Outbound_Log.IsEdiEnabled
+-- Column SQL (old): (select bp.IsEdiInvoicRecipient from C_BPartner bp where bp.C_BPartner_ID=C_Doc_Outbound_Log.C_BPartner_ID)
+-- 2024-10-10T17:47:55.979Z
+UPDATE AD_Column SET ColumnSQL='(CASE WHEN ''CS'' = (SELECT docsubtype from C_DocType where C_DocType_ID = C_Doc_Outbound_Log.C_DocType_ID) THEN (SELECT invoice.IsEdiEnabled from C_Invoice invoice where invoice.C_Invoice_ID = C_Doc_Outbound_Log.Record_ID) ELSE ((SELECT bp.IsEdiInvoicRecipient from C_BPartner bp where bp.C_BPartner_ID = C_Doc_Outbound_Log.C_BPartner_ID)) END)',Updated=TO_TIMESTAMP('2024-10-10 20:47:55','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=551507
```

Comment:
IMO we should **always** display just the value of `C_Invoice.IsEdiEnabled` IDK why the bpartner's value was displayed, but it seems clearly wrong to me.

Reply:
Changed to always take it from `C_Invoice` in case of invoices.
Regarding the `AD_SQLColumn_SourceTableColumn`, yes it does even without the `AD_SQLColumn_SourceTableColumn`. Anyway, I added a `AD_SQLColumn_SourceTableColumn` just to be sure.